programs/survival/auto_lighter.sc

```diff
@@ -47,7 +55,7 @@ spread_torches(player, initial_gamemode) ->
 		loop(4000,
 			lpos = cpos+l(rand(d), rand(d), rand(d)) - d/2;
 			if (__distance_sq(cpos, lpos) <= dd  
-					&& air(lpos) && light(lpos) < 8 && sky_light(lpos) < 8
+					&& air(lpos) && light(lpos) < global_min_light_level && (global_light_ground || sky_light(lpos) < global_min_light_level)
```

Comment:
I think ths logic is slightly flawed :/

Reply:
Hmm, it worked fine when I tested it...

Reply:
oh wait Ik what I did wrong. I forgot to invert `global_light_ground`. Because if it's true, then we want to check for the sky light thing, whereas now we do the opposite, which isn't what we want. I'd changed this on personal copy after realising, must have forgotten to transfer change here.